src/main/java/org/mule/module/xml/api/SchemaValidationException.java

```diff
@@ -33,6 +35,11 @@ public SchemaValidationException(String message, List<SchemaViolation> problems)
     this.message = Message.builder().collectionValue(problems, SchemaViolation.class).build();
   }
 
+
```

Comment:
Please add a default constructor for this class

Reply:
Added